feature/repair/src/main/java/com/iteneum/repair/data/RepairViewModel.kt

```diff
@@ -19,19 +20,27 @@ import javax.inject.Inject
  * It will handle the data work & process when loading & be done with the view.
  *
  * @author Jose Miguel Garcia Reyes
- * @author Carlos Andres Perez Hernandez
+ * @modifiedBy Carlos Andres Perez Hernandez
  */
 
 @HiltViewModel
 class RepairViewModel @Inject constructor() : ViewModel() {
     var repairRequest: RepairRequest by mutableStateOf(RepairRequest())
         private set
 
+    var mimeTypes = arrayOf("video/*", "image/*")
+
     fun getInformation() = viewModelScope.launch {
         when (response) {
             is DataState.Success -> {
                 repairRequest = response.data
             }
+            is DataState.Error -> {
+                repairRequest = RepairRequest(unitDepartment = "EEE")
+            }
+            is DataState.Loading -> {
+                repairRequest = RepairRequest(unitDepartment = "LLL")
```

Comment:
By the moment add a TODO and delete line 42

Reply:
Ok,done!